crates/api_crud/src/community/update.rs

```diff
@@ -92,70 +86,3 @@ impl PerformCrud for EditCommunity {
     send_community_ws_message(data.community_id, op, websocket_id, None, context).await
   }
 }
-
-#[async_trait::async_trait(?Send)]
-impl PerformCrud for HideCommunity {
```

Comment:
These don't need their own endpoints IMO. Consider these both community settings, which should be handled easily by the community update endpoint.

Reply:
done

Reply:
What do you think about hidecommunity also? It should probably be eliminated and also be just a part of community_update

Reply:
Oh yeah my bad.